Web/starter_project_vanguard/tailwind.config.js

```diff
@@ -9,17 +9,22 @@ module.exports = {
   theme: {
     extend: {
       colors: {
+        button_color: '#0263FF',
```

Comment:
should the button color be different from the primary color?
you can look at the design and if that is the color we are using primarily, i say turn the primary color to that